pkgs/applications/networking/protonvpn-cli-ng/default.nix

```
@@ -27,7 +27,7 @@ python3Packages.buildPythonApplication rec {
   meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
And then delete the `stdenv` parameter entirely from the top-level function call arguments (right now you have both lib and the stdenv, but you only need lib).

---

great catch!